pkg/mcs/resourcemanager/server/resource_group.go

```diff
@@ -168,6 +177,11 @@ func (rg *ResourceGroup) IntoProtoResourceGroup() *rmpb.ResourceGroup {
 			RunawaySettings:    rg.Runaway,
 			BackgroundSettings: rg.Background,
 		}
+
+		if rg.RUConsumption != nil {
+			consumption := *rg.RUConsumption
+			group.RUStats = &consumption
```

Comment:
peresit consumption both in `ResourceGroup.persistSettings` and `SaveResourceGroupStates`? I think we can only persist once in GroupState.

Reply:
This is the side effect that both grpc and persistSettings uses `IntoProtoResourceGroup` to serialize resource group. The RUStats in group setting is not used.

Reply:
So is `RuStats` useless? can we remove it from proto.ResourceGroup?

Reply:
`RuStats` is used in the grpc response